.circleci/config.yml

```
@@ -5,7 +5,7 @@ orbs:
 jobs:
   tests:
     docker: # executor type
-      - image: nipreps/miniconda:py39_2205.0
+      - image: cimg/python:3.10
```

Our miniconda image is mislabeled; it ships with 3.11 now (I will generate an appropriate label ASAP).
```suggestion
      - image: nipreps/miniconda:py39_2205.0
```

---

OK. Push forced in 0e46c3b.